### PR TITLE
List Divider (Wearable): update example and style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/953
[Problem] Example is out of dated and has wrong style.
[Solution] Update style and example.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 1]
```diff
@@ -188,9 +188,34 @@
...
+								opacity: 1;
+								content: "";
+								position: relative;
+								width: 2 * @px_base;
+								height: 49 * @px_base;
+								top: 0;
+								right: 0;
+								margin-left: 14 * @px_base;
+								margin-right: 19 * @px_base;
+								background: rgba(255, 255, 255, 0.20);
```
Comment (lslachciak) [inline src/css/profile/wearable/changeable/theme-circle/listview.less]: 1. opacity is 19% not 20% in guide
2. could you define this as color variable e.g. in src/css/profile/wearable/changeable/theme-changeable/theme.color.less

**Review by lslachciak — COMMENTED** [review]
work well, some changes requested

File: src/css/profile/wearable/changeable/theme-circle/listview.less (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/listview.less]: Done. Thanks.

**Review by lslachciak — APPROVED** [review]
looks good, work well

**Pull request merged**